### PR TITLE
Add Space Law Quick Reference to Guidebook

### PR DESCRIPTION
**Pull request by Torun-S**
## About the PR
This will add a Quick Reference guide to Space Law in the Guidebook. While by no means exhaustive of all the statutes within Space Law, this will let any player review the main rules and the criminal charges at a glance!

**Media**
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Torun
- add: Added a Space Law Quick Reference to the Guidebook

### CODE REVIEW

File: Resources/Server Info/Guidebook/Law/Charges.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+﻿<Document>
+# Charges & Sentencing
+
+This section lists all [color=#a4885c]criminal charges[/color], their [color=#a4885c]sentences[/color] as well as how to sentence a crime.
+
+## Sentencing a Crime
+- Choose one of the charges you are convicting someone of as a [color=#a4885c]Primary Charge[/color]. Add brig time between its minimum and maximum.
```
Comment (jicksaw) [inline Resources/Server Info/Guidebook/Law/Charges.xml]: Space Law says the charge with highest minimum is the primary.

File: Resources/Server Info/Guidebook/Law/Trials.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿<Document>
+# Trial Procedure
+
+On Nyanotrasen, trials are conducted in the form of an [color=#a4885c]arbitration.[/color] This means that:
+
+- You are [color=#a4885c]not required[/color] to have a [color=#a4885c]lawyer[/color].
+- You need a trial only when you are accused of a [color=#a4885c]Capital Crime[/color].
```
Comment (jicksaw) [inline Resources/Server Info/Guidebook/Law/Trials.xml:7]: > You need a trial only when you are accused of a Capital Crime.

This sounds like you have to do a trial for Capital Crimes even if the defendant doesn't want one. In the Space Law it's: "If the defendant is accused of a Capital Crime, they are entitled to trial"

**Review by jicksaw — COMMENTED** [review]
Maybe these differences are intentional, but I though I'd mention them in case they are not.

The Compound Limits and steps for sentencing clarify Space Law a lot.

File: Resources/Server Info/Guidebook/Law/Charges.xml (hunk 1 above)
Comment (Torun-S) [reply Resources/Server Info/Guidebook/Law/Charges.xml]: This is true, and will fix. The intent was to let the sentencing officer pick to be as lenient or strict as they deem fit, but I forgot that I also wanted minimums to be always observed, hence your correct comment. But since a higher minimum also corresponds to higher maximums (I just checked), this is fulfilled.

TL;DR: Will fix, thank you. Hooliganism is gonna have "Brig until compliant" as Compound Limit as well.

File: Resources/Server Info/Guidebook/Law/Trials.xml (hunk 2 above)
Comment (Torun-S) [reply Resources/Server Info/Guidebook/Law/Trials.xml:7]: **This is something I will actually transfer into full Space Law.** Trials are meant to be super quick, so there's no reason for Security to not do them. If a Capital Crime is charged, trial must happen, and that's even if the prisoner confesses, which is when it's made even speedier. It's literally three people in a room with the HoS passing judgement upon hearing an explanation of what happened. If we can't push Sec to RP that little, why bother with an MRP server. I can't dumb this down any more.

File: Resources/Server Info/Guidebook/Law/Charges.xml (hunk 1 above)
Comment (Torun-S) [reply Resources/Server Info/Guidebook/Law/Charges.xml]: Fixed with 1ea5d73.
